app/src/main/java/com/arygm/quickfix/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -1,92 +1,97 @@
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.viewinterop.AndroidView
+import com.arygm.quickfix.R
+import com.etebarian.meowbottomnavigation.MeowBottomNavigation
 
 @Composable
 fun BottomNavigationMenu(
+    selectedItem: String, // To track the selected item
     onTabSelect: (TopLevelDestination) -> Unit,
-    isUser: Boolean,
-    selectedItem: String,
+    isUser: Boolean, // Boolean flag to determine the user type
     LoD: Boolean
 ) {
-  val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
-  val selectedItemColor = if (LoD) Color(0xFF731734) else Color(0xFFB78080)
-  val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
-  val tabList: List<TopLevelDestination> =
-      if (isUser) USER_TOP_LEVEL_DESTINATIONS else WORKER_TOP_LEVEL_DESTINATIONS
 
-  NavigationBar(
-      modifier = Modifier.fillMaxWidth().height(73.dp),
-      containerColor = color1,
-      contentColor = backgroundColor) {
-        tabList.forEach { item ->
-          val isSelected = selectedItem == item.route
-          NavigationBarItem(
-              selected = selectedItem == item.route,
-              onClick = { onTabSelect(item) },
-              icon = {
-                if (isSelected) {
-                  Box(
-                      modifier =
-                          Modifier.size(width = 70.dp, height = 100.dp)
-                              .offset(y = (-20).dp)
-                              .clip(RoundedCornerShape(24.dp))
-                              .background(backgroundColor),
-                      contentAlignment = Alignment.Center) {
-                        // This box ensures the red circle is centered horizontally and offset
-                        // vertically
-                        Box(
-                            modifier =
-                                Modifier.size(37.dp)
-                                    .offset(y = (16).dp) // Moves the red circle outside the white
-                                    // ellipse
-                                    .clip(CircleShape)
-                                    .background(selectedItemColor),
-                            contentAlignment = Alignment.Center) {
-                              Icon(
-                                  imageVector = item.icon,
-                                  contentDescription = item.textId,
-                                  tint = backgroundColor)
-                            }
-                      }
-                } else {
-                  Icon(
-                      imageVector = item.icon,
-                      contentDescription = item.textId,
-                      tint = backgroundColor // Icon color when unselected
-                      )
+    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+    val selectedItemColor = if (LoD) Color(0xFF731734) else Color(0xFFB78080)
+    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
```

Comment:
color1 and backgroundColor could be more descriptive. For example, primaryColor and screenBackgroundColor would clarify their roles.